pypeit/wavecalib.py

```diff
@@ -39,28 +38,34 @@ class WaveCalib(datamodel.DataContainer):
     .. include:: ../include/class_datamodel_wavecalib.rst
 
     """
-    version = '1.0.0'
+    version = '1.1.0'
 
     # MasterFrame fun
     master_type = 'WaveCalib'
     master_file_format = 'fits'
 
     datamodel = {'wv_fits': dict(otype=np.ndarray, atype=wv_fitting.WaveFit,
                                  descr='WaveFit to each 1D wavelength solution'),
-                 'wv_fit2d': dict(otype=fitting.PypeItFit,
-                                  descr='2D wavelength solution (echelle)'),
+                 #'wv_fit2d': dict(otype=fitting.PypeItFit,
+                 #                 descr='2D wavelength solution (echelle)'),
+                 'wv_fit2d': dict(otype=np.ndarray, atype=fitting.PypeItFit,
+                                  descr='2D wavelength solution(s) (echelle).  If there is more than one, they must be aligned to the separate detectors analyzed'),
+                 'det_img': dict(otype=np.ndarray, atype=np.integer,
+                                  descr='Detector image; used occasionally by echelle.  Currently only saved if ech_separate_2d=True'),
```

Comment:
Can you add a few more words indicating what the detector image is. I.e. is it an image with an integer indicating the detector that contributed the pixel, etc.

Reply:
done

Reply:
done